opencti-platform/opencti-graphql/src/modules/workspace/workspace.graphql

```diff
@@ -88,8 +88,18 @@ input WorkspaceAddInput {
     investigated_entities_ids: [String]
 }
 
+input WorkspaceDuplicateInput {
+    type: String!
+    name: String!
+    description: String
+    manifest: String
+    tags: [String]
+    authorized_members: [MemberAccessInput!]
```

Comment:
if authorized members are reinitialized when we duplicate, do we still need to have them in the input ?

Reply:
no